otx/cli/tools/explain.py

```diff
@@ -104,7 +107,9 @@ def main():
     hyper_parameters = create(hyper_parameters)
 
     # Get classes for Task, ConfigurableParameters and Dataset.
-    if args.load_weights.endswith(".pth"):
+    if any(args.load_weights.endswith(x) for x in (".bin", ".xml", ".zip")):
+        task_class = get_impl_class(template.entrypoints.openvino)
+    elif args.load_weights.endswith(".pth"):
         if is_checkpoint_nncf(args.load_weights):
```

Comment:
BTW, I wonder if we should expect any problems when we have NNCF checkpoint.. Could you please take a look, if have time?

Reply:
Yes, I'll check it.